src/main/java/software/amazon/msk/auth/iam/CompatibilityHelper.java

```diff
@@ -0,0 +1,23 @@
+package software.amazon.msk.auth.iam;
+
+import software.amazon.awssdk.core.exception.SdkClientException;
+import software.amazon.awssdk.core.exception.SdkException;
+
+public class CompatibilityHelper {
+
+  /**
+   * Convert an exception to an SdkException
+   *
+   * @param e Exception to convert
+   * @return SdkException
+   */
+  public static SdkException toSdkException(Exception e) {
+    if (e instanceof com.amazonaws.SdkClientException) {
```

Comment:
Should be `SdkClientException` not `com.amazonaws.SdkClientException` ?

Reply:
No it is ok, we want to transforme the v1 com.amazonaws.SdkClientException to v2 software.amazon.awssdk.core.exception.SdkClientException, because some part of the code still generate the v1 exception, but another part only manage v2 exception.

It is temporary until all code is migrated